web/viewer.css

```diff
@@ -291,7 +295,7 @@ html[dir='rtl'] #toolbarSidebar {
 
 #toolbarContainer, .findbar, .secondaryToolbar {
   position: relative;
-  height: 32px;
+  height: auto;
```

Comment:
Thanks for clarifying. I've committed and pushed the change. I'll squash it shortly.